GroceryStoreRESTfullCRUD/src/main/java/com/store/controller/ChequeController.java

```diff
@@ -0,0 +1,46 @@
+package com.store.controller;
+
+import java.util.Collection;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.store.components.entity.Cheque;
+import com.store.service.ChequeService;
+
+@RestController
+@RequestMapping("/store/cheque")
```

Comment:
mapping must to fit the standard pattern for rest services version one "/rest/project/v1/method"